packages/docs/content/migration-guides/introducing-a-css-reset.mdx

```diff
@@ -0,0 +1,57 @@
+- To avoid [margin collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing), all elements will only rely on their respective top margins to better control layout. All bottom margins have been removed.
+
+## Typography
+
+### Headings
+
+All heading elements will receive the font styles defined by their parent element using the `font: inherit` rule. 
+
+[`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) is a shorthand property that encompasses several font styles, but the key thing to note is a heading will now inherit the `font-family` and `font-size` of its parent or `<body>` element. This rule ensures the semantics of a heading element are distinct from its appearance. If you need to style a heading, refer to our [headings](/foundation/typography/headings/) or [utility class](/utilities/overview/) documentation.
+
+Lastly, headings have been reset to have their bottom margins removed. All headings have a top margin set to `1.5em`.
+
+### Paragraphs
+
+Like headings, paragraphs have had their bottom margins removed and a top margin of `1em` defined. However, if a paragraph is the `:first-child` its margin will be entirely removed.
+
+### Links
+
+Links behave as they previously did, with one exception - the `text-decoration-color` is now set to `currentColor`. This change was made under the assumption that all link underlines will have the same color as its text, regardless of that text's state.
+
+## Lists
+
+The `<ol>` and `<ul>` elements have been styled to match their [`.ds-c-list`](/foundation/typography/list/) counterpart, with one key difference - they use `flex` to better control list item spacing and assist with commonly used inline list patterns.
+
+If you find yourself in need of an inline list, you can add `flex-direction: row` locally to your project. Likewise, if the spacing between list elements isn't to your liking, `gap` can be changed to the value that fits your project best.
+
+One last change worth mentioning for lists is that unstyled lists are defined under the `role="list"` attribute. This makes something like `<ul role="list">` visually the same as applying the `.ds-c-list--bare` class. Because unstyled lists have [accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type#accessibility_concerns), we opted to include this style rule to ensure proper markup is used.
+
+## Form elements
+
+Various form elements have been rebooted for simpler base styles. Here are some of the most notable changes:
+
+- The `<input>`, `<button>`, `<textarea>`, and `<select>` elements receive the font styles defined by their parent element using the `font: inherit` rule. 
+- All `:disabled` fields have `cursor: not-allowed` to prevent user confusion.
```

Comment:
Did you talk to Dan about this? Or did this stem from some other source?

Reply:
This was something Eric Bailey and I discussed during the HCM work. Mainly, disabled states typically rely on color alone to indicate what it is and adding the cursor was just an extra indicator for users. But we can get confirmation from Dan just in case.